### ISSUE TITLE
Change logic of importing global styles into components

### ISSUE THREAD

**Issue by OtisRed** [CodeForPoznan/codeforpoznan.pl_v3#235]
**General description**
Magul has a theory that we don't have to use `scoped` scss in components where we don't define local styles, since the global ones take care of everything after the import. 

**Original description**
We have a situations in multiple components, that we import global (probably) `main.scss` into scoped style for particular component:
```
<style lang="scss" scoped>
@import './../main.scss';
</style>
```

My hypothesis here is that it's unnecessary and blows our styles (as the same style exists on the global level and is added to the scoped, component level).

Things to be checked (as that's only my hypothesis):
 * styles currently consist multiple copies of original style from `main.scss`
 * we can safely remove this `scoped` tags from Vue components and they will be rendered just fine
 * we can probably import `main.scss` into JS (see: https://stackoverflow.com/a/39439413)

**Technical description**
1. One should check if Magul is right
1. If he is right all the `scoped` could be deleted from components that do not use local styles.

**Comment by MargoWM** [comment]
@Bosmanfrx We need your help!

**Comment by OtisRed** [comment]
Ok @magul I checked and removing scoping makes evil things. In principle you might have been right but I read that vuetify does not respond to that well and I can confirm that. When I was working on modal content, once I removed "scoped" all the vuetify weutify stylling was simply gone. 

Given that I don't think we are ready to give up vuetify just yet (maybe never although I actually started to think about it) I will close this task for now.
